### PR TITLE
Change UMD to be like example 'commonjsStrict.js'.

### PR DESCRIPTION
**Pull request by egilkh**
Based on suggestion from sompylasar
https://github.com/gsklee/ngStorage/pull/91#issuecomment-105325973

This does is an alternative to https://github.com/gsklee/ngStorage/pull/91

### CODE REVIEW

**Comment by momielab** [comment]
LGTM :+1:

File: ngStorage.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-        });
+        // AMD. Register as an anonymous module.
+        define(['exports', 'angular'], factory);
+    } else if (typeof exports === 'object') {
+        // CommonJS
+        factory(exports, require('angular'));
     } else {
-        return factory(angular);
+        // Browser globals
+        factory(root.ngStorage = {}, root.angular);
```
Comment (sompylasar) [inline ngStorage.js]: You get `root.ngStorage.ngStorage` having `exports.ngStorage = angular.module(...)`, I think that's not what is expected.

Comment (egilkh) [inline ngStorage.js]: Thanks @sompylasar.

I changed it in a new commit to be ```exports = angular.module(...)```.

File: ngStorage.js [hunk 2]
```diff
@@ -16,7 +19,7 @@
      * @name ngStorage
      */
 
-    angular.module('ngStorage', [])
+    exports = angular.module('ngStorage', [])
```
Comment (sompylasar) [inline ngStorage.js]: This won't work, you just re-assign the local variable, but you do not "export" anything.
Notice that in CommonJS you have to put `module.exports = something` to re-assign the property of the `module` object, not the local variable `exports` that initially refers to the same object as `module.export`.

Initially:
```
module.exports ----> { /* the exported
exports ----------->      object */ }
```

After `exports.someObject = { /* some other object */ };`
```
module.exports ----> { /* the exported
exports ----------->      object */ someObject: { /* some other object */ } }
```

After `exports = { /* some other object */ };`
```
module.exports ----> { /* the exported object */ }
exports -----------> { /* some other object */ }
```

**Pull request closed without merging**